### PR TITLE
Simple pr to add let users add a class to the markers

### PR DESCRIPTION
**Pull request by iphands**

### CODE REVIEW

File: dist/metricsgraphics.js [hunk 1]
```diff
@@ -1986,13 +1986,18 @@ function markers(args) {
...
+                })
                 .attr('stroke-dasharray', '3,1');
 
         gm.selectAll('.mg-markers')
             .data(args.markers.filter(inRange))
             .enter()
             .append('text')
-                .attr('class', 'mg-marker-text')
+                .attr('class', function (d) {
+                    return d.textclass ? 'mg-marker-text' : 'mg-marker-text ' + d.textclass;
```
Comment (wjaspers) [inline dist/metricsgraphics.js]: Shouldn't this be `return d.textclass ? 'mg-marker-text ' + d.textclass : 'mg-marker-text'`?
If textclass is not set, the current statement tries to use it, which I assume would add `undefined` to the class.

File: src/js/common/markers.js [hunk 2]
```diff
@@ -24,13 +24,18 @@ function markers(args) {
...
+                })
                 .attr('stroke-dasharray', '3,1');
 
         gm.selectAll('.mg-markers')
             .data(args.markers.filter(inRange))
             .enter()
             .append('text')
-                .attr('class', 'mg-marker-text')
+                .attr('class', function (d) {
+                    return d.textclass ? 'mg-marker-text' : 'mg-marker-text ' + d.textclass;
```
Comment (wjaspers) [inline src/js/common/markers.js]: Shouldn't this be return d.textclass ? 'mg-marker-text ' + d.textclass : 'mg-marker-text'?
If textclass is not set, the current statement tries to use it, which I assume would add undefined to the class.

**Comment by iphands** [comment]
Sorry brain fart on my end

**Comment by hamilton** [comment]
Hey @iphands - thank you for this pull request. We should have gotten to it much sooner. Would you mind resolving the conflicts and I'll merge it asap?

**Comment by iphands** [comment]
Checking this patch on my local env.

**Comment by iphands** [comment]
mg

Seems to work well on my end.

**Comment by hamilton** [comment]
Did you run gulp to get the packaged metricsgraphics.js and metricsgraphics.min.js?

**Comment by hamilton** [comment]
You know what - I'll just make the changes manually myself based on your patch. Thanks for the PR :)

**Comment by iphands** [comment]
Sorry, I thought I had it all covered.
Just so I know... is it more than a simple "gulp build:js"?

**Comment by hamilton** [comment]
No worries at all - typically if you just run 'gulp' it will also incorporate tests and such. Not sure exactly what the source of the conflict was.

**Pull request closed without merging**